### DISCUSSION TITLE
Help me delete the repo, I didn't see setting button on the repo

### DISCUSSION THREAD

**Discussion by ghost** [community/community#134586] [GitHub Education] [answered]
### Hi everyone,

I want to delete the repo from my profile, but I didn't see the setting button in the repo and there is not listing of the repo in profile view but it shows in the recent view

**Comment by AmroEid77** [comment]
First, you need to choose the repository. Then, you will see the toolbar at the top, and you need to choose the settings option. After that, you need to scroll down to the Danger Zone section, and you will find the delete repository option.
![image](https://github.com/user-attachments/assets/94343586-191a-425d-94e1-d6dc2882ea7a)
![image](https://github.com/user-attachments/assets/922514fc-e5bb-4941-a3e0-260ff14984e4)

**Comment by ghost** [comment]
![image](https://github.com/user-attachments/assets/feac063d-be21-451b-9f68-fba280e0e5e3)
There is no settings option in repo

**Reply by AmroEid77** [reply]
you don't own this repo

**Reply by ghost** [reply]
but it show in the recent view

**Reply by AmroEid77** [reply, marked as answer]
I don't think so you can delete the recent view repos
